src/game/renderer.rs

```diff
@@ -69,8 +70,10 @@ impl Renderer {
 
     /// Draws whole `LineSnake` structure
     ///
-    pub fn draw_snake(ctx: &mut Context, snake: &LineSnake) {
-        snake.body.iter().for_each(|x| x.draw(ctx));
+    pub fn draw_snake(ctx: &mut Context, snake: &Snake) {
```

Comment:
I liked foreach :)

Reply:
That's why I didn't change each and every one 😜

I can appreciate keeping a personal style to the code 😄